c2corg_api/models/document.py

```diff
@@ -304,6 +308,29 @@ def to_archive(self):
     def update(self, other):
         copy_attributes(other, self, DocumentGeometry._ATTRIBUTES)
 
+    def almost_equals(self, other):
+        g1 = None
+        if isinstance(self.geom, geoalchemy2.WKBElement):
+            g1 = geoalchemy2.shape.to_shape(self.geom)
+        else:
+            # WKT are used in the tests.
+            str1 = string.split(self.geom, ';')[1]
+            g1 = wkt.loads(str1)
+
+        g2 = None
+        if isinstance(other.geom, geoalchemy2.WKBElement):
+            g2 = geoalchemy2.shape.to_shape(other.geom)
+        else:
+            # WKT are used in the tests.
+            str2 = string.split(other.geom, ';')[1]
+            g2 = wkt.loads(str2)
+
+        # FIXME: using almost_equals with +- 0.8m
+        # OK for EPSG:3857 but what about EPSG:4326?
```

Comment:
I think the waypoints are in EPSG:3857 in the database / API and EPSG:4326 in the UI.
Will it be always like this? For all types of documents? If yes then I will change the FIXME by a comment.